resources/views/admin/pages/product/form.blade.php

```diff
@@ -0,0 +1,72 @@
+                @if($errors->has('brand_id'))
+                <span class="text-xs text-red-600 dark:text-red-400">{{ $errors->first('brand_id') }}</span>
+                @endif
+            </label>
+            <label class="block mt-4 text-sm">
+                <label for="color" class="text-gray-700 dark:text-gray-400">
+                    Select Color
+                </label>
+                <select id="color" name="color_id" class="block w-full mt-1 text-sm dark:text-gray-300 dark:border-gray-600 dark:bg-gray-700 form-select focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:focus:shadow-outline-gray" >
+                    <option value="" disabled selected>Select a Color</option>
+                    @foreach ($colors as $color)
+                    <option value="{{ $color->id }}" @if(isset($product)) {{ $product->color_id == $color->id ? 'selected' : '' }} @endif>{{ $color->name }}</option>
+                    @endforeach
+                </select>
+            </label>
+            <label class="block mt-4 text-sm">
+                <span class="text-gray-700 dark:text-gray-400">
+                    Select Sizes
+                </span>
+                <select class="block w-full mt-1 text-sm dark:text-gray-300 dark:border-gray-600 dark:bg-gray-700 form-select focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:focus:shadow-outline-gray" name="size_id">
+                    <option value="" disabled selected>Select a Size</option>
+                    @foreach ($sizes as $size )
+                    <option value="{{$size->id }}" @if(isset($product)) {{ $product->size_id == $size->id ? 'selected' : '' }} @endif>{{$size->name}}</option>
+                    @endforeach
+                </select>
+            </label>
+            <div class="mt-4 text-sm">
+                <span class="text-gray-700 dark:text-gray-400">
+                    Status
+                </span>
+                <div class="mt-2">
+                    <label class="inline-flex items-center text-gray-600 dark:text-gray-400">
+                        <input type="radio" checked @if(isset($product)) {{ $product->status == '0' ? 'checked' : '' }} @endif class="text-purple-600 form-radio focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:focus:shadow-outline-gray" name="status" value="0" />
+                        <span class="ml-2">Pending</span>
+                    </label>
+                    <label class="inline-flex items-center ml-6 text-gray-600 dark:text-gray-400">
+                        <input type="radio" @if(isset($product)) {{ $product->status == '1' ? 'checked' : '' }} @endif class="text-purple-600 form-radio focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:focus:shadow-outline-gray" name="status" value="1" />
+                        <span class="ml-2">delivered</span>
+                    </label>
+                </div>
```

Comment:
What kind of status are these? And why static?

Reply:
sir te add samye default panding  value store karva rakhyu thu

Reply:
@rajan-parmar Please comment on this

Reply:
@gauravmak I go with simple things in this PR as it's going long and after merged then I'll go for ENUM which [link](https://www.notion.so/Schedule-260b5b6bae41428d90a5576b8c200f97?p=d7fb20d331c7447d8257b9b6f84269c3&pm=s)